lib/cadet/assessments/assessment.ex

```diff
@@ -6,14 +6,17 @@ defmodule Cadet.Assessments.Assessment do
   use Cadet, :model
   use Arc.Ecto.Schema
 
-  alias Cadet.Assessments.{AssessmentType, Question, SubmissionStatus, Upload}
+  alias Cadet.Assessments.{AssessmentType, Question, SubmissionStatus, GradingStatus, Upload}
 
   schema "assessments" do
     field(:max_grade, :integer, virtual: true)
     field(:grade, :integer, virtual: true, default: 0)
     field(:max_xp, :integer, virtual: true)
     field(:xp, :integer, virtual: true, default: 0)
     field(:user_status, SubmissionStatus, virtual: true)
+    field(:grading_status, GradingStatus, virtual: true)
```

Comment:
If it's a virtual field, I don't see a reason to create migration+enum. Just plain old atom would do

Reply:
agreed!